react_native/src/screens/activity_screens/TutorsScreenComponent.js

```diff
@@ -56,10 +57,17 @@ function TutorsScreenComponent(props) {
     }
     return (
         <SafeAreaView style={[StyleSheet.absoluteFill, {paddingTop: responsiveHeight(9), marginBottom: responsiveHeight(13)}]}>
-            <SessionBookingModalComponent
-                visible={openModal}
-                closeModal={toggleModal}
-            />
+            <StripeProvider
+                publishableKey= "pk_test_51M2zHJDhRypYPdkQRZ4Cd7KIu3idER1Fz9Je6KWv7xKDdG2OENqBADizHpdPUtGX1jrEtdKvTuYJSUIeNkoKIoeM00UiSHJiq2"
```

Comment:
Nit: We have to add these to the .env on the back end side of things, but this can be done later